packages/polymath-ui/src/components/inputs/NumberInput/index.js

```diff
@@ -66,7 +66,7 @@ export class NumberInput extends Component<Props, State> {
     max: Infinity,
     maxDecimals: 18, // max amount of decimals supported by solidity
     useBigNumbers: false,
-    value: null,
+    value: '',
```

Comment:
@RafaelVidaurre you might want to take a look at this.

Reply:
Thanks for pointing that out @Grsmto . I left that change by accident